.github/workflows/canary-integration-test.yml

```diff
@@ -10,6 +10,9 @@ on:
     branches:
       - master
       - release-*
+    paths-ignore:
+      - "Documentation/**"
```

Comment:
Sure, I will add some documentation change and we can see it working.

The double start at the end was suggested in the issue description, so I tought it would be better to keep it

Reply:
> The double star

Can you look why it is needed,

Good for your understanding

Reply:
Yaa thats right,

I guess testing by adding a second commit won't help, as the current CI would be triggered what we have already in the code,

So you can remove the test commit

Reply:
I just removed it on: [3c02f40](https://github.com/rook/rook/pull/12291/commits/3c02f402b3c6664eb3769b25eba6597d3558dad9)